<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestClass</class>
 <widget class="QMainWindow" name="TestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,1" columnstretch="1,4,1">
    <item row="0" column="0" rowspan="3">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextBrowser" name="textStat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'新宋体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTreeWidget" name="treeTest">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter_4">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="btnTest">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>36</height>
          </size>
         </property>
         <property name="text">
          <string>测试</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnPause">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QListWidget" name="listNG">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>NG项列表</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>几何工具/计算刚性矩阵/testGetRigidVector</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="grpImageSrc">
      <property name="sizeIncrement">
       <size>
        <width>4</width>
        <height>3</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="QTextBrowser" name="textLog">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QComboBox" name="cboSrcInfo"/>
      <widget class="QComboBox" name="cboDstInfo"/>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGraphicsView" name="grpImageDst"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
